features/admin.server-configurations.v1/pages/connector-listing-page.tsx

```diff
@@ -73,23 +73,35 @@ export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterfa
     const isPasswordInputValidationEnabled: boolean = useSelector((state: AppState) =>
         state?.config?.ui?.isPasswordInputValidationEnabled);
 
+    const hasGovernanceConnectorReadPermission: boolean = useRequiredScopes(
+        featureConfig?.governanceConnectors?.scopes?.read
+    );
+    const hasOrganizationDiscoveryReadPermission: boolean = useRequiredScopes(
+        featureConfig?.organizationDiscovery?.scopes?.read
+    );
+    const hasResidentOutboundProvisioningFeaturePermission: boolean = useRequiredScopes(
+        featureConfig?.residentOutboundProvisioning?.scopes?.feature
+    );
+    const hasInternalNotificationSendingReadPermission: boolean = useRequiredScopes(
```

Comment:
We have used the `has{feature}ViewPermissions` convention for the feature scope check. Better to follow that IMO.